internal/pkg/plugin/resolver.go

```diff
@@ -2,11 +2,11 @@ package plugin
 
 import (
 	"fmt"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```

Comment:
Ditto moving internal import to bottom of import list.
Ditto on naming this import config_v1